lib/selectors/optimizers/advanced.js

```diff
@@ -634,6 +634,10 @@ AdvancedOptimizer.prototype.restructure = function (tokens) {
 
   var position = tokens[0] && tokens[0].kind == 'at-rule' && tokens[0].value.indexOf('@charset') === 0 ? 1 : 0;
   for (; position < tokens.length - 1; position++) {
+    if (!tokens[position] || tokens[position].kind != 'at-rule' || tokens[position].value.indexOf('@import') !== 0)
```

Comment:
@buduchail can we do it in one step together with checking for comments? Otherwise :100: